llama_stack/apis/inference/inference.py

```diff
@@ -1112,7 +1112,7 @@ async def chat_completion(
         """
         ...
 
-    @webmethod(route="/inference/batch-chat-completion", method="POST", experimental=True, level=LLAMA_STACK_API_V1)
```

Comment:
skip this, it'll cause a conflict in https://github.com/llamastack/llama-stack/pull/3261

Reply:
batch inference is gone, resolving